[SDK-165] Project Structure and Customization Guide (#14)
readyplayerme/rpm-unreal-avatar-creator: README.md

## Important

> The plugin is currently in the **alpha** stage. We recommend not using it in production until the stable version is released.

Reviewer: ```suggestion
The plugin is currently in **alpha** stage. We do not recommend using it in production until the stable version is released.
```

There are three ways you can add the **RpmAvatarCreator** plugin to your project. Add as submodules, paste the plugin inside of the **Plugins** folder, or add the blueprint version of the plugin to the Engine.

> - To add the plugin to your project as a submodule, run the following command in the terminal from your project folder. This is the preferred way of setting up the plugin.

Reviewer: ```suggestion
- To add the plugin to your project as a submodule, run the following command in the terminal from your project folder. This is the recommended way to set up the plugin.
```

> - To add the plugin to your project as a submodule, run the following command in the terminal from your project folder. This is the preferred way of setting up the plugin.

Reviewer: ```suggestion
- To add the plugin to your project as a submodule, run the following command in the terminal from your project folder. This is the recommended way of setting up the plugin.
```
